CRM/Core/Error.php

```diff
@@ -574,7 +578,15 @@ public static function debug_var($variable_name, $variable, $print = TRUE, $log
         reset($variable);
       }
     }
-    return self::debug_log_message($out, FALSE, $prefix);
+    Civi::log()->log($level, $out, ['civi.prefix' => $prefix]);
```

Comment:
So this is the biggest change in here - however it seems that Civi::log winds up calling CRM_Core_Error_Log::log

which itself calls     CRM_Core_Error::debug_log_message($message, FALSE, '', $this->map[$level]);

So it would be functionally equivalent except that the param $out  is not passed through or echo'd. I think this function might need to echo $out itself to retain that behaviour
```
  /**
   * Logs with an arbitrary level.
   *
   * @param mixed $level
   * @param string $message
   * @param array $context
   */
  public function log($level, $message, array $context = []) {
    // FIXME: This flattens a $context a bit prematurely. When integrating
    // with external/CMS logs, we should pass through $context.
    if (!empty($context)) {
      if (isset($context['exception'])) {
        $context['exception'] = CRM_Core_Error::formatTextException($context['exception']);
      }
      $message .= "\n" . print_r($context, 1);

      if (CRM_Utils_System::isDevelopment() && CRM_Utils_Array::value('civi.tag', $context) === 'deprecated') {
        trigger_error($message, E_USER_DEPRECATED);
      }
    }
    CRM_Core_Error::debug_log_message($message, FALSE, '', $this->map[$level]);
  }
```